### PR TITLE
IOS: USB fixes

### PR DESCRIPTION
**Pull request by leoetlino**
(From #8070)

A set of small fixes to the USB code in IOS HLE.

JMC47 and I have tested OH0 (microphone, Wii Speak), VEN (microphone, camera) and HIDv4 (Internet Channel, Wii Shop Channel). HIDv5 wasn't tested but is very likely to work because it is pretty much a simpler version of VEN.

This fixes *Your Shape* hanging on shutdown and camera input in *Racquet Sports*.

Note: other commits from #8070 are needed to make everything work properly on Windows.

### CODE REVIEW

**Comment by psennermann** [comment]
Worth mentioning that this PR also fixes "We Sing" and "We Sing Robbie Williams"

**Comment by leoetlino** [comment]
Yeah, this should fix all known issues with USB emulation. I'm actually surprised that *Your Shape* even worked at all given that isochronous transfer sizes were incorrectly limited...

File: Source/Core/Core/IOS/USB/OH0/OH0.cpp [hunk 1]
```diff
@@ -252,7 +252,7 @@ std::pair<ReturnCode, u64> OH0::DeviceOpen(const u16 vid, const u16 pid)
     has_device_with_vid_pid = true;
 
     if (m_opened_devices.find(device.second->GetId()) != m_opened_devices.cend() ||
-        !device.second->Attach(0))
+        !device.second->Attach())
```
Comment (BhaaLseN) [inline Source/Core/Core/IOS/USB/OH0/OH0.cpp]: Since that `if` spans multiple lines, i think we should add some curlies in here...

Comment (leoetlino) [reply Source/Core/Core/IOS/USB/OH0/OH0.cpp]: Good catch, fixed!

**Pull request merged**